tests/acceptance/features/apiAntivirus/antivirus.feature

```diff
@@ -368,3 +368,39 @@ Feature: antivirus
       | old              |
       | new              |
       | spaces           |
+
+
+    Scenario Outline: try to overwrite a file with the virus content in user share
+      Given using <dav-path-version> DAV path
+      And user "Brian" has been created with default attributes and without skeleton files
+      And user "Alice" has created folder "uploadFolder"
+      And user "Alice" has shared folder "uploadFolder" with user "Brian" with permissions "all"
+      And user "Alice" has uploaded file with content "hello" to "uploadFolder/test.txt"
```

Comment:
Can we check for both? A file in root as well as a file inside a folder, please adapt the rest of the scenario to overwrite both files and contents as well. Same for the scenario below
```suggestion
      And user "Alice" has created folder "uploadFolder"
      And user "Alice" has uploaded file with content "hello" to "uploadFolder/test.txt"
      And user "Alice" has uploaded file with content "hello" to "/test.txt"
      And user "Alice" has shared folder "uploadFolder" with user "Brian" with permissions "all"
      And user "Alice" has shared folder "/test.txt" with user "Brian" with permissions "all"
```